File: siserp-logistica-backend/src/main/java/com/besoft/siserp/lgtc/dto/request/ProductoCargarExcelDetRequest.java
package com.besoft.siserp.lgtc.dto.request;

import java.io.Serializable;

import lombok.Getter;
import lombok.Setter;
import lombok.ToString;

@Getter
@Setter
@ToString
public class ProductoCargarExcelDetRequest implements Serializable {
	private static final long serialVersionUID = 1L;

	private String marca;
	private Long idtUnidadMedida;
	private String nomUnidadMedida;
	private String codigo;
	private String nombre;
	private String descripcion;
	private Double cantidad;
	private Double precio;
}

File: siserp-logistica-frontend/src/main/siserp-logistica-frontend/src/app/modules/intranet/dto/request/producto-cargar-excel-det.request.ts
export class ProductoCargarExcelDetRequest {
    id: number;
    marca: string;
    idtUnidadMedida: number;
    nomUnidadMedida: string;
    codigo: string;
    nombre: string;
    descripcion: string;
    cantidad: number;
    precio: number;

    flgValidacion: boolean;
    validacion: string;
}